File: desafiomobileandroid/app/src/main/java/myfreecomm/br/com/desafio_mobile_android/ViewActivity/SplashActivity.java
package myfreecomm.br.com.desafio_mobile_android.ViewActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

import butterknife.BindView;
import butterknife.ButterKnife;
import myfreecomm.br.com.desafio_mobile_android.Extendables.MyFragActivity;
import myfreecomm.br.com.desafio_mobile_android.R;

public class SplashActivity extends MyFragActivity {

    private static int SPLASH_TIME_OUT = 4000;

    @BindView(R.id.imgLoad)
    ImageView imgLoad;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        ButterKnife.bind(this);

        Glide.with(this).asGif().load(R.drawable.git_load).into(imgLoad);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {

                startActivity(new Intent(SplashActivity.this, HomeActivity.class));
                overridePendingTransition(R.anim.fade_in, R.anim.fade_out);
                finish();

            }
        }, SPLASH_TIME_OUT);
    }
}

File: app/src/main/java/com/challenge/raphaelbgr/desafio/brasileirao/splash/SplashActivity.kt
package com.challenge.raphaelbgr.desafio.brasileirao.splash

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.challenge.raphaelbgr.desafio.brasileirao.R
import com.challenge.raphaelbgr.desafio.brasileirao.main.MainActivity

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }
}
